spec/lib/markdown_spec.rb

```diff
@@ -0,0 +1,190 @@
+require 'rails_helper'
```

Comment:
Let's try skip Rails here:
```suggestion
require 'spec_helper'
```